package.json

```diff
@@ -13,21 +13,26 @@
   "types": "src/types/index.d.ts",
   "scripts": {
     "banners": "./utils/maintain-banners.js",
-    "prebuild": "rimraf ./src/models ./src/factories ./src/generated-client.js",
-    "build": "okta-sdk-generator -t templates/ -o . && yarn banners",
-    "build:openapi": "openapi-generator-cli generate -i ./spec/management.yaml -g typescript -o ./src/generated  -c templates/swagger-codegen-config.json -t templates/openapi-generator && ./scripts/emitV3Types.sh",
+    "build": "yarn build:cleanGenerated && yarn build:fixSpec && yarn build:generate && yarn build:fixGenerated && yarn build:emitTypes",
+    "build:generate": "docker run --rm -v \"${PWD}:/local\" -e JAVA_OPTS=\"-Dlog.level=warn\" openapitools/openapi-generator-cli:v7.13.0 generate -i /local/spec/management.yaml -g typescript -c /local/templates/swagger-codegen-config.json -o /local/src/generated  -t /local/templates/openapi-generator --model-name-mappings User_type=UserTypeInline",
```

Comment:
Can you add a section to the README (or some other file) to document the use of docker here for builds?

Reply:
Added
https://github.com/okta/okta-sdk-nodejs/pull/438/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1108